backend/api/Controllers/UserController.cs

```diff
@@ -128,7 +133,25 @@ public IActionResult AddAccessRequest([FromBody] Model.AccessRequestModel model)
                 throw new BadRequestException("Each access request can only contain one role.");
             }
             var accessRequest = _mapper.Map<Entity.AccessRequest>(model);
-            _userService.AddAccessRequest(accessRequest);
+            _pimsService.User.AddAccessRequest(accessRequest);
+
+            // Send notification to administrators.
+            try
+            {
+                var template = _pimsService.NotificationTemplate.Get(_options.AccessRequest.NotificationTemplate);
+                var administrators = _pimsService.User.GetAdmininstrators(accessRequest.Agencies.Select(a => a.AgencyId).ToArray());
+                var notification = _pimsService.NotificationQueue.GenerateNotification(
+                    String.Join(";", administrators.Select(a => a.Email).Concat(new[] { _options.AccessRequest.SendTo }).Where(e => !String.IsNullOrWhiteSpace(e))),
+                    template,
+                    new AccessRequestNotificationModel(accessRequest, _options));
+                await _pimsService.NotificationQueue.SendNotificationsAsync(new[] { notification });
```

Comment:
A notification will be sent out, but in a non production environment it will go to the user who submitted it...  I'll think this through and see how I can override and override...